### DISCUSSION TITLE
add customChangelogUrl handlebar templating support

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#24852] [Suggest an Idea]
### What would you like Renovate to be able to do?

Many docker image packages are lacking the  [OCI label org.opencontainers.image.source](https://github.com/opencontainers/image-spec/blob/main/annotations.md) to point to source and have their release notes in the created PR, as expected by [Docker datasource](https://docs.renovatebot.com/modules/datasource/#docker-datasource).

The preferred approach is to contribute the LABEL instruction to each DockerFile project, (e.g. https://github.com/kudobuilder/kuttl/pull/391) or suggest upstream projects to do the update in mass (e.g. https://github.com/crossplane/crossplane/issues/3370)

However, this approach takes elapsed time to pay off.

As a complementary workaround, many docker images are named after their github source repo, therefore the customChangelogUrl can easily be programmatically constructed from the docker image {{depName}} if the customChangelogUrl pakcage rule would support handlebar templating

See related discussion at https://github.com/renovatebot/renovate/discussions/18280 for further details

### If you have any ideas on how this should be implemented, please tell us here.

Similar to other package rules handlebar templating support

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Feel free to contribute to https://github.com/renovatebot/renovate/blob/8ee28a44921dced5074ac994fefbe0a3862dd592/lib/modules/datasource/metadata-manual.ts#L67 too in the meantime
